### ISSUE TITLE
Hover/display option to show ALT text of audio and video media attachments

### ISSUE THREAD

**Issue by rmoriz** [mastodon/mastodon#35789]

**Comment by nykula** [comment]
Inability to view alt text for audio should be reclassified as a bug because it's a serious accessibility and usability issue.
